docker-compose.yml

```diff
@@ -20,10 +21,12 @@ services:
   db:
     container_name: db
```

Comment:
Remove this as it creates a global container name that will conflict with any staging instance running on the same host. It doesn't need to be set in order to refer to the service name in a configuration.